pkg/operator/apis/monitoring/v1/node_types_test.go

```diff
@@ -68,11 +68,29 @@ func TestValidateClusterNodeMonitoring(t *testing.T) {
 			fail:        true,
 			errContains: "scrape timeout 2s must not be greater than scrape interval 1s",
 		},
+		{
+			// Regression test for https://github.com/GoogleCloudPlatform/prometheus-engine/issues/479
+			desc: "Duplicated job name",
+			eps: []ScrapeNodeEndpoint{
+				{
+					Interval: "10s",
+				},
+				{
+					Interval: "10000ms",
+				},
+			},
+			fail:        true,
+			errContains: "/r1/metrics for endpoints with index 0 and 1. Consider creating a separate resource for each job",
+		},
 	}
 
 	for _, c := range cases {
 		t.Run(c.desc+"", func(t *testing.T) {
 			nm := &ClusterNodeMonitoring{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "r1",
+					Namespace: "ns1",
```

Comment:
chore: ClusterNodeMonitoring should not have a namespace.

Reply:
Yup, but it has such field, so wanted to ensure we don't rely on this. Happy to remove.